.github/workflows/sdk-build-container.yml

```diff
@@ -12,25 +12,24 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-name: SDK - Build Base Container
+name: SDK - App Builder
```

Comment:
Maybe App Builder is a bit misleading, maybe something with App-Baseimage?

Reply:
It is only used as "builder image" for the build stage of app images which I wanted to visualize here. "from ...app_builder as builder"

Reply:
```suggestion
name: Create App Builder image
```
?